PaymentHighway/UI/TextField.swift

```diff
@@ -176,21 +174,24 @@ open class TextField: UITextField, UITextFieldDelegate {
     }
     
     private func updateTextImage() {
-        guard let textFieldType = textFieldType,
-              theme.textImages.contains(textFieldType) else { return }
-        
-        if let oldImage = self.viewWithTag(imageTag) {
-            oldImage.removeFromSuperview()
-        }
-        if let image = theme.textImageView(textFieldType: textFieldType, height: self.frame.height) {
-            paddingX = self.frame.height
-            image.tag = imageTag
-            addSubview(image)
+        guard let textFieldType = textFieldType else { return }
+        if let image = theme.textImageView(textFieldType: textFieldType) {
+            // text-image padding half of the padding
+            let paddingImageText = theme.textPaddingX*0.5
```

Comment:
Normal Swift style is to have spaces around arithmetic operands, so `theme.textPaddingX * 0.5` could be used? This applies to multiple places in the source, but no strong opinions about this.